addons/portal/controllers/mail.py

```diff
@@ -91,12 +103,16 @@ def _message_post_helper(res_model, res_id, message, token='', nosubscribe=True,
 class PortalChatter(http.Controller):
 
     @http.route(['/mail/chatter_post'], type='http', methods=['POST'], auth='public', website=True)
-    def portal_chatter_post(self, res_model, res_id, message, **kw):
+    def portal_chatter_post(self, res_model, res_id, message, attachment_ids=None, **kw):
         url = request.httprequest.referrer
-        if message:
+        if attachment_ids:
+            attachment_ids = literal_eval(attachment_ids)
+            if isinstance(attachment_ids, int):
+                attachment_ids = [attachment_ids]
+        if message or attachment_ids:
             # message is received in plaintext and saved in html
             message = plaintext2html(message)
-            _message_post_helper(res_model, int(res_id), message, **kw)
+            _message_post_helper(res_model=res_model, res_id=int(res_id), message=message, attachment_ids=attachment_ids, **kw)
```

Comment:
Note, not really linked to pr but we need to filter kw here to avoid injections in message post.

Reply:
@Xavier-Do ,  we already filter some kw in `_message_post_helper`. Thanks